### PR TITLE
Update name of GetSpatialIdsWithinRadiusOfLine

### LINKED ISSUE #18: Rename GetSpatialIdsWithinRadiusOfLine
**Describe the bug**
GetSpatialIdsWithinRadiusOfLine does not return spatial ids but extended spatial ids.
The name is wrong.

**Expected behavior**
Please rename GetSpatialIdsWithinRadiusOfLine.
For example, GetExtendedSpatialIdsWithinRadiusOfLine

**Additional context**
[Add Auxiliary SpatialID functions](https://github.com/trajectoryjp/spatial_id_go/pull/6)

### PR DESCRIPTION
**Pull request by buscandoaverroes**
- Fixes #18 
- Replace GetSpatialIdsWithinRadiusOfLine with GetExtendedSpatialIdsWithinRadiusOfLine
- Passes package tests

### CODE REVIEW

File: transform/voxel_around_line.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	geodesy "github.com/trajectoryjp/geodesy_go/coordinates"
 )
 
-// GetSpatialIdsWithinRadiusOfLine
+// GetExtendedSpatialIdsWithinRadiusOfLine
 // 直線と線からradiusの距離以内の拡張空間IDを取得する
 //
 // 引数：
```
Comment (HarutakaMatsumoto) [inline transform/voxel_around_line.go:23]: Append the comment of skipsMeasurement.

File: transform/voxel_around_line.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ import (
 //	拡張空間IDスライス： []string
 //  error: エラー
 
-func GetSpatialIdsWithinRadiusOfLine(startPoint *object.Point, endPoint *object.Point, radius float64, hZoom int64, vZoom int64, skipsMeasurement bool) ([]string, error) {
+func GetExtendedSpatialIdsWithinRadiusOfLine(startPoint *object.Point, endPoint *object.Point, radius float64, hZoom int64, vZoom int64, skipsMeasurement bool) ([]string, error) {
```
Comment (HarutakaMatsumoto) [inline transform/voxel_around_line.go:36]: This change is incompatible.
Change over the module name from v2 to v3.

**Review by HarutakaMatsumoto — CHANGES_REQUESTED** [review]
I reviewed this.
Check it out.

File: transform/voxel_around_line.go (hunk 2 above)
Comment (buscandoaverroes) [reply transform/voxel_around_line.go:37]: resolved here https://github.com/trajectoryjp/spatial_id_go/pull/19/commits/faaea39b58feb9951abf3ab1d1e1bb0df056b5c6

File: transform/voxel_around_line.go (hunk 1 above)
Comment (buscandoaverroes) [reply transform/voxel_around_line.go:23]: resolved here https://github.com/trajectoryjp/spatial_id_go/pull/19/commits/9431a3af06191bc384802142950fd0d3eae8b174

**Comment by buscandoaverroes** [comment]
- in `/examples/checkZoom` passes `go run main.go`
- passes build check on github

**Comment by HarutakaMatsumoto** [comment]
I reviewed and merged this.
Thank you for your contributions!

**Pull request merged**